app/controllers/medicaid/amounts_income_controller.rb

```diff
@@ -18,19 +18,23 @@ def income_attributes
 
     def employed_monthly_income_attributes
       {}.tap do |income_hash|
-        current_application.number_of_jobs&.times do |count|
+        employed_number_of_jobs&.times do |count|
           income_hash["employed_monthly_income_#{count}"] =
             current_application.employed_monthly_income[count]
         end
       end
     end
 
+    def employed_number_of_jobs
+      current_application.primary_member.employed_number_of_jobs
+    end
+
     def skip?
-      not_employed? && not_self_employed? && not_receiving_unemployment?
+      nobody_employed? && not_self_employed? && not_receiving_unemployment?
     end
 
-    def not_employed?
-      !current_application&.employed?
+    def nobody_employed?
+      !current_application&.anyone_employed?
```

Comment:
Was just mentioning to @hartsick that it might make it easier on us to consider implementing these as first class methods (ie: `#no_one_employed?`) on the current_application instead of negating the predicate (not anyone employed).

On that note - if we continue to do that and see that there are a LOT of predicate methods defined on the application object we should think about extracting them to some sort of policy object.

Reply:
is the main reason for implementing these that it will reduce duplication or is there an additional reason?

Reply:
Adding methods like `no_one_employed`? I suggest that purely for ease of comprehension. It's easier for me to unpack "no one is employed" instead of "not anyone employed".  The "no one" versus "not anyone" namely - just an english / legibility thing.